nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -68,6 +68,7 @@ BtNavigator::BtNavigator()
   declare_parameter("global_frame", std::string("map"));
   declare_parameter("robot_base_frame", std::string("base_link"));
   declare_parameter("odom_topic", std::string("odom"));
+  declare_parameter("bt_activate_groot_monitoring", true);
```

Comment:
Default should be false, we don't want extra traffic most users don't need

Reply:
How about `enable_groot_monitoring`? Shorter, more to the point

Reply:
approve + done